cli/test/test-min-size-gate.mjs

```
@@ -0,0 +1,80 @@
+#!/usr/bin/env bun
+// Behaviour test for the startup size gate (MinSizeGate).
+//
+// Below the floor it must show a resize prompt (naming the short dimension) and
+// hide the wizard; at/above the floor it must render the wizard content through
+// unchanged. Rendered via the real VT grid so we assert what the terminal shows.
+import { Box, Text } from 'ink'
+import process from 'node:process'
```

ESLint

[error] Expected "node:process" (value-builtin) to come before "ink" (value-external).

(perfectionist/sort-imports)